deps.edn

```diff
@@ -284,10 +284,13 @@
   :cljs
   {:extra-paths ["test" "shared/test"]
    :extra-deps
-   {cljs-bean/cljs-bean     {:mvn/version "1.9.0"}
-    com.lambdaisland/glogi  {:mvn/version "1.2.164"}
-    io.github.metabase/hawk {:sha "45ed36008014f9ac1ea66beb56fb1c4c39f8342b"}
-    thheller/shadow-cljs    {:mvn/version "2.20.20"}}}
+   {cider/cider-nrepl             {:mvn/version "0.30.0"}
+    cider/piggieback              {:mvn/version "0.5.3"}
+    cljs-bean/cljs-bean           {:mvn/version "1.9.0"}
+    com.lambdaisland/glogi        {:mvn/version "1.2.164"}
+    io.github.metabase/hawk       {:sha "45ed36008014f9ac1ea66beb56fb1c4c39f8342b"}
```

Comment:
Are you including the Hawk test runner just for stuff like `=?` ?

Reply:
It's not new in this PR, just moved around.

I don't really remember anymore. It came in the with the `metabase.util` port, I think because some of those tests use `=?` etc.